bundles/automation/org.eclipse.smarthome.automation.provider.file/src/main/java/org/eclipse/smarthome/automation/internal/provider/file/WatchServiceUtil.java

```diff
@@ -25,15 +25,15 @@
 @SuppressWarnings("rawtypes")
 public class WatchServiceUtil {
 
-    static Map<AbstractFileProvider, Map<String, AutomationWatchService>> WATCH_SERVICES = new HashMap<AbstractFileProvider, Map<String, AutomationWatchService>>();
+    static Map<AbstractFileProvider, Map<String, AutomationWatchService>> watchServices = new HashMap<AbstractFileProvider, Map<String, AutomationWatchService>>();
```

Comment:
This should also be `private final static Map<...> WATCH_SERVICES` as it is only used in this util class. Further this util class is not supposed to be extended by inheritance.